ui/lib/kubernetes/addon/components/page/configuration.hbs

```diff
@@ -9,7 +9,7 @@
     <InfoTableRow @label="Kubernetes host" @value={{@config.kubernetesHost}} />
     {{#if @config.kubernetesCaCert}}
       <InfoTableRow @label="Certificate">
-        <CertificateCard @certificateValue={{@config.kubernetesCaCert}} />
+        <CertificateCard @data={{@config.kubernetesCaCert}} @isPem={{true}} />
```

Comment:
Changed this arg away from `certificateValue` since this is used for more than certificates (used for private keys too). Naming it `data` seems to be most in line with [these docs](https://developer.hashicorp.com/vault/api-docs/secret/pki#generate-certificate-and-key)